cypress/integration/visual-testing/homepage-test.spec.js

```diff
@@ -0,0 +1,41 @@
+import { baseURL, eyesOpen } from '../../utils';
+describe('Visual  Regression Tests', () => {
+  beforeEach(() => {
+    cy.visit(baseURL);
+    eyesOpen('Home Page');
+  });
+  afterEach(() => {
+    cy.eyesClose();
+  });
+
+  it('should display sign up options', () => {
+    cy.get('#navigation-butt').click();
+    cy.get('#navigation-butt').should('be.visible');
```

Comment:
Cypress' `click` command will only work if the button for the given selector is visible so you don't need to add this check. For reference see https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Actionability